contracts/contracts/sbtc-token.clar

```diff
@@ -107,17 +94,17 @@
 					sender: principal, 
 					to: principal, 
 					memo: (optional (buff 34)) })))
-	(fold complete-individual-transfer recipients (ok u0))
+	(fold transfer-many-iter recipients (ok u0))
```

Comment:
This uses the same naming pattern as for protocol-mint-many

Reply:
Copy, works.